public/scripts/menu.js

```diff
@@ -14,24 +14,34 @@ if (typeof cssmenu_no_js === 'undefined') {
             open_main_item = container.classList.contains('open') ? container: null;
         }
         e.stopPropagation();
-        return false;
+        e.preventDefault();
     }
 
     // menu button for mobile devices
     var dom_hamburger = document.body.querySelector(".hamburger.expand-toggle");
-    dom_hamburger.addEventListener('click', function(evt){
-        return handleMenuClick(dom_hamburger.parentNode, evt, true);
+    dom_hamburger.addEventListener('click', function(e){
+        handleMenuClick(dom_hamburger.parentNode, e, true);
+    });
+
+    // search container narrow screens
+    var dom_search = document.body.querySelector(".search-container > .expand-toggle");
+    dom_search.addEventListener('click', function(e){
+        handleMenuClick(dom_search.parentNode, e, false);
+        document.body.querySelector("#search-query > input").focus();
```

Comment:
focusing works fine on my phone without this - out of interest: what is your behavior without this addition?